pkg/generated/bindata.go

```diff
@@ -324,6 +478,153 @@ func translationsKubectlEn_usLc_messagesK8sPo() (*asset, error) {
+msgid "Apply a configuration to a resource by filename or stdin"
+msgstr ""
+"Appliquer une configuration à une ressource par nom de fichier ou stdin"
+
+# https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/config/config.go#L39
+msgctxt "Modify kubeconfig files"
+msgid "Modify kubeconfig files"
+msgstr "Modification des fichiers kubeconfig"
+
+# https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/config/create_authinfo.go#L103
+msgctxt "Sets a user entry in kubeconfig"
+msgid "Sets a user entry in kubeconfig"
+msgstr "Définit une entrée utilisateur dans kubeconfig"
+
+# https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/config/create_cluster.go#L67
+msgctxt "Sets a cluster entry in kubeconfig"
+msgid "Sets a cluster entry in kubeconfig"
+msgstr "Définit une entrée de cluster dans kubeconfig"
+
+# https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/config/create_context.go#L57
+msgctxt "Sets a context entry in kubeconfig"
+msgid "Sets a context entry in kubeconfig"
+msgstr "Définit une entrée de contexte dans kubeconfig"
+
+# https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/config/current_context.go#L48
+msgctxt "Displays the current-context"
+msgid "Displays the current-context"
+msgstr "Affiche le contexte actuel"
+
+# https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/config/delete_cluster.go#L38
+msgctxt "Delete the specified cluster from the kubeconfig"
+msgid "Delete the specified cluster from the kubeconfig"
+msgstr "Supprimer le cluster spécifié à partir du kubeconfig"
+
+# https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/config/delete_context.go#L38
+msgctxt "Delete the specified context from the kubeconfig"
+msgid "Delete the specified context from the kubeconfig"
+msgstr "Supprimer le contexte spécifié à partir du kubeconfig"
+
+# https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/config/get_clusters.go#L40
+msgctxt "Display clusters defined in the kubeconfig"
+msgid "Display clusters defined in the kubeconfig"
+msgstr "Clusters d'affichage définis dans le kubeconfig"
+
+# https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/config/get_contexts.go#L62
+msgctxt "Describe one or many contexts"
+msgid "Describe one or many contexts"
+msgstr "Décrire un ou plusieurs contextes"
+
+# https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/config/set.go#L59
+msgctxt "Sets an individual value in a kubeconfig file"
+msgid "Sets an individual value in a kubeconfig file"
+msgstr "Définit une valeur individuelle dans un fichier kubeconfig"
+
+# https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/config/unset.go#L47
+msgctxt "Unsets an individual value in a kubeconfig file"
+msgid "Unsets an individual value in a kubeconfig file"
+msgstr "Désactive une valeur individuelle dans un fichier kubeconfig"
```

Comment:
`Désactive` literally means `disable`. I am not aware of a clear antonym in French for `Set / Define / Other synonyms` (nor are the dictionaries). I'd personally use `Delete`.

```
Supprime une valeur individuelle dans un fichier kubeconfig
```